components/secret_storage/secret/orm.py

```diff
@@ -0,0 +1,67 @@
+"""SQLAlchemy's schemas for the secrets database."""
+
+from datetime import datetime
+from typing import Optional
+
+from cryptography.fernet import Fernet
+from secret_storage.secret import models
+from sqlalchemy import DateTime, MetaData, String
+from sqlalchemy.orm import DeclarativeBase, Mapped, MappedAsDataclass, mapped_column
+from ulid import ULID
+
+metadata_obj = MetaData(schema="secrets")  # Has to match alembic ini section name
+
+
+def encrypt_string(key: bytes, data: str) -> str:
+    """Encrypt a given string."""
+    return Fernet(key).encrypt(data.encode()).decode()
```

Comment:
ideally we would use a salt, so if someone has the key they can't just decrypt everything easily.
https://cryptography.io/en/latest/fernet/#using-passwords-with-fernet has an example.
We just need to pick a good salt,like maybe user id for user secrets, group id for group secrets etc.